Servers/UI/OJS.Servers.Ui/ClientApp/src/components/submissions/submissions-list/SubmissionsList.tsx

```diff
@@ -49,67 +49,77 @@ const SubmissionsList = ({
     const submissionBtnClass = 'submissionBtn';
     const submissionsTypeLabelClassName = concatClassNames(styles.submissionTypeLabel);
 
-    const renderSubmissionListItem = useCallback((submission: ISubmissionDetails) => {
-        const { id: selectedSubmissionId } = selectedSubmission || {};
-        const {
-            id,
-            points,
-            maximumPoints,
-            isProcessed,
-            createdOn,
-            submissionType,
-            isOfficial,
-        } = submission;
-        const isSelectedSubmission = id === selectedSubmissionId;
-        const selectedClassName = isSelectedSubmission
-            ? styles.selected
-            : '';
+    const renderSubmissionListItem = useCallback(
+        (submission: ISubmissionDetails) => {
+            const { id: selectedSubmissionId } = selectedSubmission || {};
+            const {
+                id,
+                points,
+                maximumPoints,
+                isProcessed,
+                createdOn,
+                submissionType,
+                isOfficial,
+            } = submission;
+            const isSelectedSubmission = id === selectedSubmissionId;
+            const selectedClassName = isSelectedSubmission
+                ? styles.selected
+                : '';
 
-        const itemClassName = concatClassNames(
-            styles.submissionContainer,
-            selectedClassName,
-        );
+            const itemClassName = concatClassNames(
+                styles.submissionContainer,
+                selectedClassName,
+            );
 
-        const buttonState = isSelectedSubmission
-            ? ButtonState.disabled
-            : ButtonState.enabled;
+            const buttonState = isSelectedSubmission
+                ? ButtonState.disabled
+                : ButtonState.enabled;
 
-        const typeLabelText = isOfficial
-            ? ContestParticipationType.Compete
-            : ContestParticipationType.Practice;
+            const typeLabelText = isOfficial
```

Comment:
I think theres a helper method for this. If not, please move this into a common logic file